populate_DB/seed.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs')
+const db = require('../db/db')
+
+
+
+let file = fs.readFileSync('animal_list.txt')
+
+animal_array = []
```

Comment:
please declare it with const and camelCase and row 10 can be a direct assignment